src/sprout/Helpers/Fb.php

```diff
@@ -608,11 +608,11 @@ public static function chunkedUpload($name, array $attrs = [], array $params = [
                 $view->orig_file = ['name' => $file['original'], 'size' => filesize($temp_path)];
                 $type = File::getType($file['original']);
 
-            // Existing file stored on disk
+            // Existing file stored on disk. Let the file backend handle finding the info
             } else if ($file) {
-                $temp_path = $opts['file_url'] ?? File::baseDir() . $file;
+                $temp_path = $file;
                 $view = new PhpView('sprout/file_confirm');
-                $view->orig_file = ['name' => 'Existing file', 'size' => @filesize($temp_path)];
+                $view->orig_file = ['name' => 'Existing file', 'size' => File::size($temp_path)];
```

Comment:
I bet it wasn't fun finding all of these.

Reply:
Insert <tedious> gif here.